src/table/newgrf_debug_data.h

```diff
@@ -272,11 +272,38 @@ static const NIFeature _nif_industrytile = {
 /*** NewGRF industries ***/
 
 static const NIProperty _nip_industries[] = {
-	NIP(0x10, Industry, produced_cargo[0], NIT_CARGO, "produced cargo 0"),
-	NIP(0x10, Industry, produced_cargo[1], NIT_CARGO, "produced cargo 1"),
-	NIP(0x11, Industry, accepts_cargo[0],  NIT_CARGO, "accepted cargo 0"),
-	NIP(0x11, Industry, accepts_cargo[1],  NIT_CARGO, "accepted cargo 1"),
-	NIP(0x11, Industry, accepts_cargo[2],  NIT_CARGO, "accepted cargo 2"),
+	NIP(0x10, Industry, produced_cargo[ 0], NIT_CARGO, "produced cargo 0"),
```

Comment:
Not sure what the exact purpose of this is, apart from being something with NewGRF. Is it correct to just extend like this?

Reply:
Based on discussions about NewGRF support on IRC, this is certainly wrong.

Reply:
This is mostly a debug GUI. Putting all cargos here makes the GUI bigger than needed, but otherwise it does not affect other stuff.